docs/xshareclassesdisableonrestore.md

```diff
@@ -0,0 +1,54 @@
+﻿<!--
+* Copyright (c) 2017, 2023 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -Xshareclasses:disableOnRestore
+
+**(Linux&reg; x86, Linux on POWER&reg; (Little Endian), and Linux on IBM Z&reg; only)**
+
+The `-Xshareclasses` option is an existing option and `disableOnRestore` is its new parameter that can be used if the CRIU feature is available. This option is disabled by default.
+
+:fontawesome-solid-triangle-exclamation:{: .warn aria-hidden="true"} **Restrictions:**
+
+- This option takes effect only when the `-XX:+EnableCRIUSupport` option is enabled.
+- This option can be used only on restore.
+
+## Parameters
+
+### `disableOnRestore`
+
+        -Xshareclasses:disableOnRestore
+
+The `-Xshareclasses:disableOnRestore` option disables further use of the shared classes cache (SCC) on restore. You can still use the shared classes utility APIs, for example, to obtain metrics about the cache, however the VM behaves as if the cache is empty.
```

Comment:
> You can still use the shared classes utility APIs.

I suggest to add URL of the utility APIs (https://www.eclipse.org/openj9/docs/shrc/#the-java-shared-classes-utility-api) so that it is easier for users to know what we are talking about.